### PR TITLE
Client style

### PR DESCRIPTION
**Pull request by JordanLindley**
Was originally meant to be a branch to style the notes app and ended up rounding out CRUD actions.
Create Note button has been implemented, posts to db and displays properly.
Auto-save functionality implemented using debounce.
Refactored some of the server setup, fixed SQL queries to return results for JSON parsing.

Will open a new branch that is actually for stlying this time.

### CODE REVIEW

File: src/client/App.tsx [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { useEffect, useState } from 'react';
 import NoteBook from './components/NoteBook';
 import Note from './components/Note';
 import Delete from './components/Delete';
+import { json } from 'body-parser';
+import { stringify } from 'querystring';
+import { create } from 'domain';
```
Comment (ScottLindley) [inline src/client/App.tsx]: I'm not sure any of these new imports are used?

File: src/client/App.tsx [hunk 2]
```diff
@@ -36,16 +40,56 @@ function App() {
...
     <div className="App">
-      <NoteBook notes={notes} onSelectNote={selectNote} />
-      {selectedNote ? (
-        <div className="display-container">
-          <Note note={selectedNote} />
-          <Delete noteId={selectedNote.id} onDeleteNote={deleteNote}></Delete>
-        </div>
-      ) : null}
+      <div id="ui">
+        {/* <Create> */}
```
Comment (ScottLindley) [inline src/client/App.tsx]: Can probably remove this comment now.

**Review by ScottLindley — APPROVED** [review]
Mostly looks great, just a few comments about stuff I think we can remove.

**Comment by JordanLindley** [comment]
Removed comment and unused imports.

**Pull request merged**